enumcover.go

```diff
@@ -160,3 +174,22 @@ func allConstsWithType(pass *analysis.Pass, targetType string) []constVal {
 	})
 	return consts
 }
+
+func buildConstMap(pass *analysis.Pass, targetType string) map[string]string {
```

Comment:
It seems like we shouldn't need both this and `allConstsWithType`, they 're doing the same thing. Can we just use this map?

Reply:
I agree, although I have a limited understanding of how the maps is loaded, considering the different typeNames. So I was being careful, not to break anything. I will give it a try.